a11y-input/src/ally-input.js

```diff
@@ -37,6 +37,8 @@ export class A11yInput extends LitElement {
   }
 
   set value (newValue) {
+    debugger; // debugging Type Error
```

Comment:
### error found! The shadow DOM has not rendered before calling the setter.

```js
<a11y-input>
  #shadow-root (open)
</a11y-input>
```

### Dev Tools screenshot:
<img width="756" alt="Screen Shot 2020-03-18 at 3 25 13 PM" src="https://user-images.githubusercontent.com/10120600/77004243-c7377780-692c-11ea-91ff-1355e4b1f2bd.png">

```js
$ console.log(this); // run in Chrome Dev Tools console 
```

Reply:
### Setter (before update):

```js
set value (newValue) {
    debugger; // debugging Type Error

    if (newValue === 'cat') {
      console.log('We like cats too. 🐱');
    }

    this.inputEl.value = newValue;
  }
```